packages/@react-spectrum/s2/style/spectrum-theme.ts

```diff
@@ -513,7 +528,11 @@ export const style = createTheme({
       base: colorToken('background-base-color'),
       'layer-1': colorToken('background-layer-1-color'),
       'layer-2': weirdColorToken('background-layer-2-color'),
-      pasteboard: weirdColorToken('background-pasteboard-color')
+      pasteboard: weirdColorToken('background-pasteboard-color'),
+      'focus-ring': {
+        default: colorToken('focus-indicator-color'),
+        forcedColors: 'Highlight'
+      }
```

Comment:
wanted the focus ring color that only exists in outline styles as the background color for the nubbin and resizer handle

Reply:
I don't love this being part of the theme, could you use a variable instead to get the focus-ring color and then set the background color to that variable?

Reply:
sure, mainly did this for any other cases where the background color of an element needed to match the same blue as the focus ring/outline but I'll move it to a variable

Reply:
I don't think that's going to be common outside of components, but if it is, then we should probably get it added to spectrum itself